catalogue/webapp/components/WorkDetails/WorkDetails.js

```diff
@@ -29,6 +29,27 @@ import { clientSideSearchParams } from '@weco/common/services/catalogue/search-p
 import Download from '../Download/Download';
 import PaletteSimilarityBox from '../PaletteSimilarityBox/PaletteSimilarityBox';
 
+type TimeProps = {|
+  hours: string,
+  minutes: string,
+  seconds: string,
+|};
+
+function doubleDigits(n) {
+  return n > 9 ? `${n}` : `0${n}`;
+}
+
+function convertMS(milliseconds: number): TimeProps {
```

Comment:
Nitpick – milliseconds is one word, so don't feel like the 's' deserves capping up. I'd also consider spelling it out (and possibly adding what they're being converted to). Triple nit-pick, if you like!